cmd/filters/remove_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2024 The K8sGPT Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filters
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRemoveCmd(t *testing.T) {
+	require.Equal(t, "remove", removeCmd.Name())
+
+	err := removeCmd.Args(&cobra.Command{}, []string{"arg1"})
+	require.NoError(t, err)
+
+	err = removeCmd.Args(&cobra.Command{}, []string{"arg1", "arg2"})
+	require.ErrorContains(t, err, "accepts 1 arg(s), received 2")
+
+	// Set the configuration file in viper
+	configFileName := "delete-config.json"
+	data := map[string]interface{}{
```

Comment:
I think so, we need to create and set a Viper configuration file because this is from where k8sgpt cli fetches the latest filters when the user adds, removes, or lists them.